coins/ethereum/README.md

```diff
@@ -0,0 +1,23 @@
+# Ethereum
+
+This package contains Ethereum-related functionality, specifically deploying and interacting with Serai contracts.
+
+## Requirements
+
+- npm / node.js (16.0+)
```

Comment:
Related to the submodule comment about moving solely to Rust.

Reply:
done!